datareduction/datareducer.cpp

```diff
@@ -539,22 +531,6 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
          ));
          continue;
       }
-      if(*it == "HallE") {
```

Comment:
Can this be substituted with the fsgrid equivalent instead?

Reply:
TODO